zuul.d/pipelines.yaml

```diff
@@ -1055,3 +1055,29 @@
       github:
         status: 'failure'
       mysql:
+
+- pipeline:
+    name: arm_comment_only
+    description: |
+      This pipeline is triggered by github comment only for ARM test. Before CI
```

Comment:
You'd better to tell us commenting What to trigger this pipeline. Or keep the same style with other pipelines in description.